src/data/gen2Banner-urls.ts

```diff
@@ -10,12 +10,40 @@ export const GEN2BANNER_URLS = [
   '/vue/',
   '/react-native/',
   '/javascript/how-amplify-works/',
+  '/angular/how-amplify-works/',
+  '/react/how-amplify-works/',
+  '/nextjs/how-amplify-works/',
+  '/vue/how-amplify-works/',
   '/javascript/how-amplify-works/capabilities/',
```

Comment:
All the sub pages under /how-amplify-works/ can be removed (including for JS) as this was changed in the final publication and these are headings rather than individual pages now - can remove anything with:
 /how-amplify-works/capabilities/
  /how-amplify-works/building-an-app/
 /how-amplify-works/frontend-hosting/
 /how-amplify-works/connect-to-aws/
 /how-amplify-works/support-matrix/
 /how-amplify-works/amplify-architecture/